src/openklant/migration/v1/data.py

```diff
@@ -177,11 +179,21 @@ def set_from_external_subject(self, subject_data: dict) -> None:
                 self.subject_type = subject_class.klant_type
                 return
 
-    def to_digitaal_adres(self) -> DigitaalAdres | None:
+    def to_digitaal_adres_email(self) -> DigitaalAdres | None:
         if not self.emailadres:
             return
 
-        return DigitaalAdres(adres=self.emailadres)
+        return DigitaalAdres(
+            adres=self.emailadres, soort_digitaal_adres=SoortDigitaalAdres.email
+        )
+
+    def to_digitaal_adres_phonenumber(self) -> DigitaalAdres | None:
+        if not self.telefoonnummer:
+            return
+        return DigitaalAdres(
```

Comment:
I see in the dataclass the default for omschrijving is `Emailadres`, could you remove that default?

<img width="801" height="171" alt="image" src="https://github.com/user-attachments/assets/89b70ea2-9e1d-48fd-af3a-fa4f101e37d7" />